keras/preprocessing/image.py

```diff
@@ -963,6 +963,9 @@ def _recursive_list(subpath):
         for fname in files:
             is_valid = False
             for extension in white_list_formats:
+                if fname.lower().endswith('.tiff'):
```

Comment:
This will warns a tremendous amount times, isn't it?
I would just do a check before returning.

Reply:
Actually, I am pretty sure it won't (I also ran it). 

See the docs at http://docs.python.org/2/library/warnings.html:

"Repetitions of a particular warning for the same source location are typically suppressed."

Reply:
Good then!